react-docs/src/pages/Views.tsx

```diff
@@ -0,0 +1,202 @@
+import React, { useMemo } from 'react';
+import { MainHeading } from './atoms';
+import CodePreview from './CodePreview';
+import { interval } from 'rxjs';
+import { take, map } from 'rxjs/operators';
+import { useStream, NOT_YET_EMITTED, connect } from 'rxbeach/react';
+
+const Views = () => {
+  return (
+    <div>
+      <MainHeading>Views</MainHeading>
+      <p>Goal: Simple views that react to state changes</p>
+      <p>
+        A view's core responsibility is to render HTML and dispatch events
```

Comment:
Strictly speaking we are not using HTML, just 'display data' ?  Or display data with web technologies? Technically it's more about creating a DOM?